draft-vandijk-dprive-ds-dot-signal-and-pin/draft-vandijk-dprive-ds-dot-signal-and-pin.md

```diff
@@ -171,7 +176,7 @@ echo example.com. IN DNSKEY 0 3 225 MIICIj...AAQ== \
 example.com.	3600	IN	DS	7573 225 2 fcb6...c26c
 ```
 
-[TODO: what if a server has different keys depending on crypto algorithm negotiation? probably need some words on that somewhere, perhaps not (only) in this section]
+[TODO: what if a server has different keys depending on crypto algorithm negotiation? probably need some words on that somewhere, perhaps not (only) in this section. Most likely, operators SHOULD not to do that at all - this is a new protocol, servers don't have to deal with Windows XP clients]
```

Comment:
This would delay future new TLS algorithm rollouts. However, I think this can be solved by simply adding more DS records (if necessary temporarily). However, this procedure is actually not different from the normal TLSA procedure, so I do not think we necessarily need to mention anything about it? Its actually the same as a key rollover.

Reply:
Could be a very long key rollover if an operator does decide to support two TLS algorithms in parallel. If this is likely to be a pitfall in generating DSes, we should have words on it. If we hold the "don't do that then" position, we can remove the words.

I did not check - do the TLSA RFCs say anything about 'protocol agility'?

Reply:
Robin mentioned that https://tools.ietf.org/html/rfc6698#section-5 at least mentions roll overs.

Reply:
Multiple algorithms is covered by #32, I removed the text here.